server/routes/v1/requests.ts

```diff
@@ -43,6 +44,7 @@ export const getRequests = [
     const requests = await readRequests({
       type: type as RequestTypes,
       filter: filter === 'all' ? undefined : req.user._id,
+      archived: !!archived && archived === 'true',
```

Comment:
I think I wrote this bit, but it can be simplified to `archived: archived === 'true',`

Reply:
fixed